src/test/java/com/iexec/core/task/TaskTestsUtils.java

```diff
@@ -36,9 +36,19 @@ public class TaskTestsUtils {
     public final static String TEE_TAG = TeeUtils.TEE_SCONE_ONLY_TAG; //any supported TEE tag
     public final static String RESULT_LINK = "/ipfs/the_result_string";
 
-    public static Task getStubTask(long maxExecutionTime) {
-        Task task = new Task(CHAIN_DEAL_ID, 0, DAPP_NAME, COMMAND_LINE, 1, maxExecutionTime, NO_TEE_TAG);
+    public static Task getStubTask() {
+        Task task = new Task(CHAIN_DEAL_ID, 0, DAPP_NAME, COMMAND_LINE, 1, 60000, NO_TEE_TAG);
```

Comment:
maybe to be uniform with the other method, putting a final here would be nice.

Reply:
fixed